catalyst/callbacks/metrics/sklearn.py

```diff
@@ -0,0 +1,34 @@
+from typing import Callable, Dict, Mapping
+
+import torch
+
+from catalyst.callbacks.metric import FunctionalBatchMetricCallback
+from catalyst.core.runner import IRunner
+from catalyst.metrics import FunctionalBatchMetric
+
+
+class SklearnCallback(FunctionalBatchMetricCallback):
```

Comment:
is it possible to also add loader-based sklearn metrics into the separate Callback?
we could not compute AUCs in per-batch mode

Reply:
nevertheless, we could just filter the loader-based metrics during metric_fn check for now
to simplify the PR

Reply:
should I filter by name?.. or how?